docs/identities/sign-in/two-step-registration.mdx

```diff
@@ -5,30 +5,16 @@ sidebar_label: Two-step registration
 slug: two-step-registration
 ---
 
-With a one-step registration, when you enable multiple authentication strategies (for example password, code via email, and
-passkey), the identity traits will be repeated for each authentication strategy.
+[Identity traits](../../kratos/manage-identities/managing-users-identities-metadata#traits) are data associated with an identity
+that can be modified by the user. The traits are configured through the identity schema.
 
-Compare this to a two-step registration: Now, the user is prompted for the identity traits in the first step, and asked to choose
-a credential method for authentication next. This results in a more stream-lined user experience.
+With a one-step registration - also called _unified registration_ - the identity traits will be repeated for each authentication
```

Comment:
Should we just called it `With unified registration (formerly one-step legacy registration)`

and

`With profile-first registration (formerly two-step registration)`

and at some point remove the old naming?

Reply:
ok that makes sense!

Reply:
fixed